feathr_project/feathr/akv_client.py

```diff
@@ -0,0 +1,22 @@
+from azure.keyvault.secrets import SecretClient
+from azure.identity import DefaultAzureCredential
+from loguru import logger
+from azure.core.exceptions import ResourceNotFoundError
+
+class akvClient:
```

Comment:
[nitpick] - Suggest changing it to more descriptive name AzureKeyVaultClient, the way we have for PurviewClient etc.